src/components/ActionCard.tsx

```diff
@@ -93,21 +63,29 @@ const ActionCard: React.FC<IProps> = ({
 
   return (
     <Card to={to} className={className} hideArrow={true}>
-      <ImageWrapper
-        isRight={isRight}
-        isBottom={isBottom}
+      <Flex
+        minH={"260px"}
+        bg={"cardGradient"}
+        direction={"row"}
+        justify={isRight ? "flex-end" : "center"}
+        align={isBottom ? "flex-end" : "center"}
         className="action-card-image-wrapper"
+        boxShadow="inset 0px -1px 0px rgba(0, 0, 0, 0.1)"
```

Comment:
question: is this the correct way to use `boxShadow`? It works but, we are sort of being oddly specific in this case

Reply:
It is ok for now.
We are going to clean them up once we finish with the entire migration & we implement the [open design system](https://github.com/ethereum/ethereum-org-website/issues/6284).